src/injected/element-based-view-model-creator.ts

```diff
@@ -48,7 +48,9 @@ export class ElementBasedViewModelCreator {
         return this.getSelectorToVisualizationMap(
             cardSelectionData,
             results,
-            'platformInfo' in storeData && storeData.platformInfo ? storeData.platformInfo : null,
+            storeData && 'platformInfo' in storeData && storeData.platformInfo
```

Comment:
nit: could this be replaced with `storeData?.platformInfo || null`

Reply:
Since this could be assessment store data the typechecker requires us to do the in check, so I think it has to stay this way unfortunately :(